admin/schema.php

```diff
@@ -884,6 +884,27 @@
 
 # Release marker: 1.3.0
 
+# Count / occurrences / votes 
+$g_upgrade[210] = array( 'AddColumnSQL', array( db_get_table( 'bug' ), "
+	votes					I		NOTNULL DEFAULT '1'",
+	) );
+$g_upgrade[211] = array( 'CreateIndexSQL', array( 'idx_bug_count', db_get_table( 'bug' ), 'votes' ) );	
+
+# A standalone table is required for "alias", because not all DB allow multiple NULLs in UNIQUE index
+$g_upgrade[212] = array('CreateTableSQL', array( db_get_table( 'bug_alias' ), "
```

Comment:
Actually, I was trying to avoid a join to render a issues.  If this is just used at crash reporting time, that's fine.  I assume the count for each issue will be cached on the bug table.